test/libsolidity/syntaxTests/double_variable_declaration_050.sol

```diff
@@ -0,0 +1,13 @@
+// double_variable_declaration_050.sol
```

Comment:
In the long run, we probably need another level of directories, but it might be fine like this for now.